awx/main/expect/isolated_manager.py

```diff
@@ -238,7 +224,7 @@ def check(self, interval=None):
                 idle_timeout=remaining,
                 job_timeout=remaining,
                 pexpect_timeout=5,
-                proot_cmd=self.proot_cmd
+                proot_cmd='bwrap'
```

Comment:
We're running the isolated playbooks under bwrap, which _probably_ isn't necessary, but meh.  It also doesn't hurt.